thirdparty/build-thirdparty.sh

```diff
@@ -1239,6 +1239,20 @@ build_clucene() {
     fi
 }
 
+# avx2ki
+build_avx2ki() {
+    check_if_source_exist $AVX2KI_SOURCE
+    cd $TP_SOURCE_DIR/$AVX2KI_SOURCE
+    rpm2cpio boostkit-ksl-2.0.0-1.aarch64.rpm | cpio -id
+    mv usr/local/ksl $TP_INSTALL_DIR
```

Comment:
need to make sure $TP_INSTALL_DIR/ksl doesn't exist before doing the move operation. Otherwise it will fail for second wrong (in case of retry).